codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/protocols/eventstream/ClientEventStreamMarshallerGeneratorTest.kt

```
@@ -77,6 +77,16 @@ class ClientEventStreamMarshallerGeneratorTest {
 
                 let mut request = rx.expect_request();
 
+                let headers = request.headers();
+                assert!(headers
+                    .get("content-type")
+                    .map(|hdr| hdr == ${rpcEventStreamTestCase.inner.requestContentType.dq()})
+                    .unwrap_or(true));
```

Should both of these be `.unwrap_or(false)`? Seems like the spec says these should always be set, so we probably want to fail if they are unset?